content/04.results.md

```diff
@@ -1 +1,195 @@
+The "Class" annotation column denotes the binarized set definition of each country, either "Cited" or "Mentioned".
+Panel E shows the top 15 words extracted from articles mentioning the countries depicted in Panel D, with the most significant proportional frequency between the two defined country sets.
+The width of the bar depicts the log10(Frequency in Mentions / Frequency of Citations).
+](https://github.com/nrosed/nature_news_disparities/raw/09df81eac6d8450127552f0b9dd3a6c1212b10eb/figure_notebooks/tmp_files/fig4_tmp/fig4_main.png "Figure 4"){#fig:fig4 width=6in}
+
+
+After identifying name origin differences in cited and quoted persons, we wanted to determine if countries 1) are represented at difference rates 2) vary in how scientific content is related to a country.
+To do this analysis, we used three sources of information 1) country-related entities mentioned in the news text, 2) country affiliations of cited authors in news articles, 3) country affiliation of authors in Nature and Springer.
+Figure @fig:fig4 a shows example input data and a schematic of the analysis.
+We provide further processing details in [Methods].
+First, we interrogated the country affiliations of cited authors.
+We assigned an affiliation to a manuscript if any author has affiliation from a specific country. 
+Therefore a single manuscript may have multiple country affiliations.
+We analyzed any possible country affiliation for an article due to limits imposed by the Springer API.
+Affiliation query results from the Springer API return all country affiliations for a specific manuscript and are not linked to one particular author.
+ 
+We analyzed a total of XX articles and considered all authors within the article and their affiliations for this analysis.
+We found that most cited articles have at least one author with an affiliation within the United States, followed by the United Kingdom, Germany, and France (Figure @fig:fig4 b).
+Interestingly, we found a strong citation over-enrichment of many top-cited countries, but we found no evidence of under-enrichment by East Asian countries (Figure @fig:fig4 c, Supplemental Figure 4 a).
+Next, we were interested in identifying content differences between countries or groups of countries.
+For example, we wanted to identify if a country was the subject (i.e., their scientific policies, environment, pollution) or whether the science within a nation was the subject.
+To do this, we need to calculate the mention rate of each country.
+To identify if a country was mentioned in an article, we took all organizations, countries, states, or provinces identified by coreNLP's named entity tagger.
+We then took all the identified region-related named entities and utilized OpenStreetMap to get the affiliated country.
+Since there may be errors in both coreNLP and OpenStreetMap, a country is only assumed to be mentioned when at least two unique entities map to the same country in a single article.
```

Comment:
```suggestion
Since there may be errors in both coreNLP and OpenStreetMap, we only assumed a country was mentioned when at least two unique entities mapped to the same country in a single article.
```